### PR TITLE
[14.0][MIG] shopinvader_validation_token from v10.0

### PR DESCRIPTION
**Pull request by acsonefho**
Migration of `shopinvader_validation_token`

Please also review pr from v10.0: https://github.com/shopinvader/odoo-shopinvader/pull/843

### CODE REVIEW

**Comment by lmignon** [comment]
@acsonefho Can you rebase and fix the tests so we can move forward. Thank you!

**Comment by acsonefho** [comment]
@shopinvader/shopinvader-maintainers

**Comment by acsonefho** [comment]
@shopinvader/shopinvader-maintainers Can you review and merge please?

File: shopinvader_validation_token/data/mail_template.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            name="email_from"
+        >${(object.shopinvader_backend_id.company_id.email or '')|safe}</field>
+        <field name="subject">Security token</field>
+        <field name="email_to">${object.email|safe}</field>
+        <field name="model_id" ref="model_shopinvader_security_token" />
+        <field name="auto_delete" eval="False" />
+        <field name="lang">${object.lang}</field>
+        <field
+            name="body_html"
+        ><![CDATA[ Your token: ${object.token} valid until ${format_tz(object.expiration_datetime)} ]]></field>
```
Comment (HviorForgeFlow) [inline shopinvader_validation_token/data/mail_template.xml]: format_tz is not available on Odoo 14.0:

```
odoo.exceptions.UserError: Failed to render template : 'format_tz' is undefined - - -
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Functional review: It is not possible to retrieve security token email

File: shopinvader_validation_token/data/mail_template.xml (hunk 1 above)
Comment (acsonefho) [reply shopinvader_validation_token/data/mail_template.xml]: @HviorForgeFlow Thanks for your review. I just fix it.
The `format_tz` is no more available in `v14.0`

**Review by rousseldenis — APPROVED** [review]
Code review.

@acsonefho Could you check why tests are failing ?

**Comment by rousseldenis** [comment]
@Cedric-Pigeon @acsonefho IMHO, in order to avoid logic in shopinvader, this should deserve its own module in OCA/server-auth.

**Comment by acsonefho** [comment]
> @acsonefho Could you check why tests are failing ?

Always the same issue in queue.

https://github.com/OCA/queue/pull/440

**Comment by sebastienbeau** [comment]
can yoy rebase so we can merge it ?

**Comment by acsonefho** [comment]
> can yoy rebase so we can merge it ?

It's green :green_circle: . Can you merge please? :pray: @sebastienbeau

**Comment by sebastienbeau** [comment]
/ocabot merge patch
(I am doing a merge sprint) so do not hesitate to ping me today

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by acsonefho** [comment]
@sebastienbeau Can you retry merge please?

**Pull request merged**